pages.cn/linux/apk.md

```diff
@@ -0,0 +1,27 @@
+# apk
+
+> Alpine linux的包管理工具.
+
+- 从所有的远程仓库中更新仓库索引
```

Comment:
I believe a colon is needed at the end of this line, at least in English. Is this not the case in Chinese?

Reply:
yes, there should be a colon, it's my fault. I have corrected the error